libnet/src/libnet_if_addr.c

```diff
@@ -386,7 +437,7 @@ libnet_select_device(libnet_t *l)
                 "%s(): no network interface found", __func__);
         return (-1);
     }
-	
+
```

Comment:
are you stripping trailing whitespace? I approve, but its cosmetic, better put in another commit (or leave alone).